src/main/java/Chatbot.java

```diff
@@ -0,0 +1,145 @@
+import java.io.*;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+import java.time.LocalDateTime;
+import java.time.format.FormatStyle;
+import java.util.ArrayList;
+import java.util.List;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.temporal.ChronoUnit;
+
+public class Chatbot {
+    public String line = "____________________________________________________________";
+    public List<Task> list = new ArrayList<>();
+
+    public void greeting() {
+        System.out.println(line);
+        System.out.println("Hello! I'm Duke\n" +
+                "What can I do for you?");
+        System.out.println(line);
+    }
+
+    public void addTask(Task t) {
+        System.out.println(line);
+        System.out.println("Got it. I've added this task:");
+        System.out.println(t.printTask());
+        System.out.println("Now you have " + list.size() + (list.size() > 1 ? " tasks" : " task") + " on the list");
+        System.out.println(line);
+    }
+    public void generateList() {
+        System.out.println(line);
+        System.out.println("Here are the tasks in your list:");
+        for (int i = 0; i < list.size(); i++) {
+            Task t = list.get(i);
+            System.out.println((i + 1) + ". " + t.printTask());
+        }
+        System.out.println(line);
+    }
+
+    public void delete(int i) {
+        System.out.println(line);
+        System.out.println("Noted. I've removed this task:");
+        System.out.println(list.get(i - 1).printTask());
+        list.remove(i - 1);
+        System.out.println("Now you have " + list.size() + (list.size() > 1 ? " tasks" : " task") + " on the list");
+        System.out.println(line);
+    }
+
+    public boolean chat(String s) throws IncorrectInputException, IOException {
```

Comment:
Maybe extract the components in this method into different methods? There is a lot of information in just 1 method.